### ISSUE TITLE
Cannot import styles.css

### ISSUE THREAD

**Issue by gokselgoktas** [radix-ui/themes#46]
Trying to import `styles.css` via `@import url(@radix-ui/themes/styles.css);` in a CSS file results in the following error:

```
ERROR in ./src/app/index.css (./node_modules/css-loader/dist/cjs.js!./src/app/index.css)

Module build failed (from ./node_modules/css-loader/dist/cjs.js):
Error: Package path ./styles.css is not exported from package [...]\node_modules\@radix-ui\themes
(see exports field in [...]\node_modules\@radix-ui\themes\package.json)
```

However, when I go check the `exports` field in the package's `package.json`, I can clearly see the `./styles.css` being exported.

Similarly, importing `styles.css` in a TypeScript (or a JavaScript) file via `import '@radix-ui/themes/styles.css'` results in a no-op. It simply does not inject the CSS into the generated chunks and has no effect whatsoever on the HTML output. Importing any other CSS file works as expected.

I am using Webpack 5.88.2.
Any idea what's going on? Could this be a bug in Webpack?

**Comment by anjankarmakar10** [comment]
The leading cause of this error is that styles.css does not exist in radix-ui-themes. the @radix-ui/themes/ point to radix-ui-themes folder and you try to access the styles.css like that @radix-ui/themes/styles.css, but there are no styles.css file in this folder.

**Comment by anjankarmakar10** [comment]
The leading cause of this error is that styles.css does not exist in radix-ui-themes. the @radix-ui/themes/ point to radix-ui-themes folder and you try to access the styles.css like that @radix-ui/themes/styles.css, but there are no styles.css files in this folder.

**Comment by anjankarmakar10** [comment]
The leading cause of this error is that styles.css does not exist in radix-ui-themes. the @radix-ui/themes/ point to radix-ui-themes folder and you try to access the styles.css like that @radix-ui/themes/styles.css, but there are no styles.css files in this folder. I also noticed you ignored the styles.css in the .gitignore file '/packages/radix-ui-themes/styles.css'

**Comment by gokselgoktas** [comment]
@anjankarmakar10 I don't think this is relevant at all. You're looking at the source code of the package and not the final package being distributed. If you do `npm i @radix-ui/themes` you'd notice that it results in a package called `@radix-ui` under `node_modules`, which then contains a `themes` submodule with `styles.css` in it as expected. So, I don't think this is the issue here.

**Comment by Tanya9kin** [comment]
Same here! 
I tried importing the @radix-ui/themes/styles.css in my index.js file, imported the <Theme> component and wrapped my main <App> component with it, and nothing shows up after building. 
There is no CSS file in my dist directory, I didn't even get an error about this, just saw that the project does not display any styles.

**Comment by gokselgoktas** [comment]
@Tanya9kin For now the solution was for me to directly import the `styles.css` file from under `node_modules` in a local CSS file and then import that in my `index.js`:

```
@import url(../../node_modules/@radix-ui/themes/styles.css);
```

**Comment by Tanya9kin** [comment]
@gokselgoktas  Thanks! it worked!

**Comment by thursdaybw** [comment]
Is this issue a duplicate of: https://github.com/radix-ui/themes/issues/59

This issue sounds very much the same, except you guys are using url() in the imports, and have not specified which environment, that other issue is that it doesnt' work in plain react, but does in next js.

**Comment by vladmoroz** [comment]
Can you confirm whether this is resolved in 2.0.0?

**Comment by Dev-Squid** [comment]
Yeah this happend to me recently when running dev all was looking good but when deploying or running prod build it was not picking up some CSS from radix-ui. 

To fix this I imported the css into the global.css. File looks like this

`import url(../node_modules/@radix-ui/themes/styles.css);
@tailwind base;
@tailwind components;
@tailwind utilities;


:root {
  --foreground-rgb: 0, 0, 0;
}

@media (prefers-color-scheme: dark) {
  :root {
    --foreground-rgb: 255, 255, 255;
    --background-start-rg`
